tests/api/api/test_projects.py

```diff
@@ -130,6 +131,54 @@ def test_redirection_from_worker_to_chief_delete_project(
                 assert response.text == expected_response
 
 
+def test_get_project_function_when_using_not_normalize_name(
```

Comment:
Looking at the logic you are testing I think this need to be under the `tests_functions` and on the DB layer, as you are not actually testing any "API" logic over here but mainly that the `db` layer is able to return a function with an underscore.